packages/regcool.vm/regcool.vm.nuspec

```
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>regcool.vm</id>
-    <version>2.0.0.20240408</version>
+    <version>2.0.0.20240410</version>
```

```suggestion
    <version>0.0.0.20240410</version>
```
As we don't know which tool version we are installing (potentially 3)